lib/utils/manager/download_manager/download_manager.dart

```diff
@@ -26,14 +26,14 @@ class DownloadManager {
 
   final workingQueue = getIt.get<DownloadWorkerQueue>();
 
-  final Map<String, DownloadFileInfo> _eventIdMapDownloadFileInfo = {};
+  final Map<String, DownloadFileInfo> eventIdMapDownloadFileInfo = {};
```

Comment:
Why need to change from private to public?

Reply:
If you change it to public, remember that the external can change its value, be careful with that

Reply:
done